test/controllers/admin/comfy/users_controller_test.rb

```diff
@@ -76,6 +76,20 @@ class Comfy::Admin::UsersControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
+  test "#edit: show forum button if selected " do
+    sign_in(@user)
+    get admin_users_url(subdomain: @domain)
+    @restarone_subdomain.update(forum_enabled: true)
```

Comment:
@Gloria-kuang both cases will pass regardless if navigation items are present 
<img width="1386" alt="Screen Shot 2022-06-16 at 3 08 15 PM" src="https://user-images.githubusercontent.com/35935196/174146915-3f6a9011-bbd5-4fcc-8997-5465497acda1.png">

Reply:
to see pages in the application visit

``` txt
lvh.me:5250/rails/info/routes

```